frontend/views/containers/proposals/ProposalItem.vue

```diff
@@ -309,7 +309,7 @@ export default ({
         if (
           this.currentGroupState._vm.invites?.[inviteKeyId]?.status === INVITE_STATUS.VALID &&
           this.currentGroupState._vm.authorizedKeys?.[inviteKeyId] &&
-          this.currentGroupState._vm.authorizedKeys[inviteKeyId]._notAfterHeight != null &&
+          this.currentGroupState._vm.authorizedKeys[inviteKeyId]._notAfterHeight == null &&
```

Comment:
Oof, seems like an important bug catch. Surprised tests didn't catch this

Reply:
Indeed. Well, this breaks the tests on this branch, but for some reason not on master. The previous line is wrong anyhow, as `_notAfterHeight != null` means that the invite has since been revoked.